stock_picking_batch_extended/__manifest__.py

```diff
@@ -20,10 +21,9 @@
         "views/report_batch_picking.xml",
         "views/stock_picking_views.xml",
         "views/stock_warehouse.xml",
+        "views/stock_picking.xml",
         "views/res_config_settings_views.xml",
         "wizard/stock_picking_to_batch_views.xml",
     ],
-    "installable": True,
-    "post_init_hook": "post_init_hook",
```

Comment:
Why removing this?

Reply:
This comment is pending

Reply:
@BorisFL

Reply:
@BorisFL Could you restore post_init_hook ? Thanks

Reply:
Yes, I have just recovered it